### DISCUSSION TITLE
Windows static library

### DISCUSSION THREAD

**Discussion by rallyordie** [magiblot/tvision#81] [Q&A] [answered]
Fantastic project! Are there pre-built Win32 static libraries available? Many thanks.

**Comment by magiblot** [comment]
Hi! No, but I can add them!

**Comment by magiblot** [comment, marked as answer]
You can now find pre-compiled static libraries in the Artifacts section of a GitHub Actions run, as in https://github.com/magiblot/tvision/actions/runs/3569708665 (the `library-x64` and `library-x86` files, at the bottom).

However, please note that these static libraries are built with the `/MT` compiler flag (*use the static version of MSVC's run-time library*). In order to link against them, your program will have to be compiled with that flag as well.

**Comment by rallyordie** [comment]
Many thanks!

	-----------------------------------------From: "magiblot" 
To: "magiblot/tvision"
Cc: "rallyordie", "Author"
Sent: Monday November 28 2022 7:16:17PM
Subject: Re: [magiblot/tvision] Windows static library (Discussion
#81)

	You can now find pre-compiled static libraries in the Artifacts
section of a GitHub Actions run, as in 
[1]https://github.com/magiblot/tvision/actions/runs/3569708665 [2]
(the library-x64 and library-x86 files, at the bottom). 

	However, please note that these static libraries are built with the
/MT compiler flag (_use the static version of MSVC's run-time
library_). In order to link against them, your program will have to be
compiled with that flag as well. 

	—
 Reply to this email directly,  view it on GitHub [3], or  unsubscribe
[4].
 You are receiving this because you authored the thread.Message ID:  

Links:
------
[1] https://github.com/magiblot/tvision/actions/runs/3569708665
[2] https://github.com/magiblot/tvision/actions/runs/3569708665
[3]
https://github.com/magiblot/tvision/discussions/81#discussioncomment-4258450
[4]
